govcd/api_vcd_test.go

```diff
@@ -331,7 +332,7 @@ func (vcd *TestVCD) infoCleanup(format string, args ...interface{}) {
 // Gets the two components of a "parent" string, as passed to AddToCleanupList
```

Comment:
```suggestion
 // Gets the two (or three) components of a "parent" string, as passed to AddToCleanupList
```

Reply:
Not exactly like that, but I altered it with another request from Linas.